docs/core-manage-asdf-vm.md

```diff
@@ -277,29 +315,226 @@ asdf update --head
 brew upgrade asdf
 ```
 
+### -- Pacman --
+
+Refer to the [Arch Pacman upgrade documentation](https://wiki.archlinux.org/index.php/pacman#Upgrading_packages).
+
```

Comment:
Same as above, must either download a new `PKGBUILD` and rebuild it, or have an AUR helper do it for you.